hopp/sites/site_info.py

```diff
@@ -82,6 +84,7 @@ def __init__(self, data,
             #. ``tz``: int (optional), timezone code (metadata purposes only) [-]
             #. ``no_solar``: bool (optional), if ``True`` solar data download for site is skipped, otherwise solar resource is downloaded from NSRDB
             #. ``no_wind``: bool (optional), if ``True`` wind data download for site is skipped, otherwise wind resource is downloaded from wind-toolkit
+            #. ``no_wave``: bool (optional), if ``True`` wave data download for site is skipped, otherwise wind resource is downloaded from wind-toolkit
```

Comment:
I didn't think you were getting wave data from the wind toolkit. Can you update the source comment?